### PR TITLE
Adding Bala Hantsi

### PR DESCRIPTION
**Pull request by bhantsi**
All requirements checked

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -29,6 +29,32 @@ module.exports = [
       'CSS',
     ],
   },
+  {
+    name: 'Bala Hantsi',
+    description:
+      'Software developer, full stack developer (in training), lately focused on frontend. Enjoy traveling and gaming.',
+    url: 'https://github.com/bhantsi/uses',
```
Comment (gatarelib) [inline src/data.js:36]: Looks like the page is not found

**Review by gatarelib — APPROVED** [review]

**Review by gatarelib — CHANGES_REQUESTED** [review]
Page not found.

**Comment by bhantsi** [comment]
I need more clarification please, I see all URLs has /uses

**Comment by BlakeCampbells** [comment]
@bhantsi the URL you've provided redirects to a GitHub 404 page. Maybe it's setup as a private page

**Comment by bhantsi** [comment]
All done, page can be now be found

**Review by BlakeCampbells — APPROVED** [review]

**Pull request merged**